src/libraries/Microsoft.PowerFx.Core/Public/ErrorKind.cs

```diff
@@ -36,7 +36,8 @@ public enum ErrorKind
         Numeric = 24,
         InvalidArgument = 25,
         Internal = 26,
-        NotApplicable = 27,
+        NotApplicable = 27,
+        Overflow = 28,          // Decimal TODO: fold into Numeric?  Odd error message in canvas for IfError(1e308*10,FirstError.Message) about non-finite number?  Where is Excel's #Value?
```

Comment:
this seems reasonable to me. 
You'll need to update the EnumStore as well with this.